README.md

```diff
@@ -21,17 +21,17 @@ bundle exec jekyll serve --watch
 
 The `--watch` means that you will not have to restart the server to see changes reflected on your locally running site. However, you always need to restart if you make changes to `_config.yml`.
 
-The output of this command will tell you where the site is running locally. It will be something like `localhost:4000/$conference_year/` (note the closing slash).
+The output of this command will tell you where the site is running locally. It will be something like `localhost:4000/$conference_year_slug/` (note the closing slash).
 
-`$conference_year` is whatever is set as the `baseurl` in [_config.yml](_config.yml).
+`$conference_year_slug` is whatever is set as the `baseurl` in [_config.yml](_config.yml).
```

Comment:
Not this, the `baseurl` stays the same

Reply:
Ah, `conference_year_slug` isn't used anywhere now so we don't actually need it in `_config.yml` at all any more, `baseurl` is enough. I'll remove it, and also update the README to just refer to `baseurl` in all these places - I think that's clearer.